Compile errors in THmovie.cpp from #1140 merge

The code doesn't compile for me, there are two errors:
````
~/corsix-th/CorsixTH/Src/th_movie.cpp:1044:6: error: prototype for 'void THMovie::play(const SDL_Rect&, int)' does not match any in class 'THMovie'
 void THMovie::play(const SDL_Rect &destination_rect, int iChannel)

In file included from ~/corsix-th/CorsixTH/Src/th_movie.cpp:23:0:
~/corsix-th/CorsixTH/Src/th_movie.h:228:10: error: candidate is: void THMovie::play(int)
     void play(int iChannel);
````
and
````
~/corsix-th/CorsixTH/Src/th_movie.cpp:1056:6: error: prototype for 'void THMovie::refresh()' does not match any in class 'THMovie'
 void THMovie::refresh() {}

In file included from ~/corsix-th/CorsixTH/Src/th_movie.cpp:23:0:
~/corsix-th/CorsixTH/Src/th_movie.h:253:10: error: candidate is: void THMovie::refresh(const SDL_Rect&)
     void refresh(const SDL_Rect &destination_rect);
````

My configuration:
```` bash
cmake -D WITH_MOVIES:BOOL=OFF \
      -D SDL_LIBRARY:PATH=/usr/lib64/libSDL2.so \
      -D SDL_INCLUDE_DIR=/usr/include/SDL2 \
      -D CMAKE_BUILD_TYPE=DEBUG \
      .
````
@ecrips : Did you handle all video libraries?

--- a/CorsixTH/Src/th_movie.cpp
+++ b/CorsixTH/Src/th_movie.cpp
@@ -1041,7 +1041,7 @@ void THMovie::setRenderer(SDL_Renderer *pRenderer) {}
 bool THMovie::moviesEnabled() const { return false; }
 bool THMovie::load(const char* filepath) { return true; }
 void THMovie::unload() {}
-void THMovie::play(const SDL_Rect &destination_rect, int iChannel)
+void THMovie::play(int iChannel)
 {
     SDL_Event endEvent;
     endEvent.type = SDL_USEREVENT_MOVIE_OVER;
@@ -1053,7 +1053,7 @@ int THMovie::getNativeWidth() const  { return 0; }
 bool THMovie::hasAudioTrack() const  { return false; }
 const char* THMovie::getLastError() const { return nullptr; }
 void THMovie::clearLastError() {}
-void THMovie::refresh() {}
+void THMovie::refresh(const SDL_Rect &destination_rect) {}
 void THMovie::allocatePictureBuffer() {}
 void THMovie::deallocatePictureBuffer() {}
 void THMovie::readStreams() {}

